src/main/features/event-management/views/eventManagement.njk

```diff
@@ -181,7 +181,7 @@
         <p class="govuk-body">{{ data.subsection4_title4_descriiption }}</p>
        
        
-        {% if status == 'PUBLISHED' %}
+        {% if status == 'published' %}
```

Comment:
Please convert status to lower case because sometimes it will come in upper case then our condition will false.